nl/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -1,9 +1,9 @@
 <!ENTITY pbMacroSearchPanel.width "600">
 
 <!ENTITY insertMacro.key "l">
-<!ENTITY insertMacro.modifiers "accel">
+<!ENTITY insertMacro.modifiers "versnellen">
 
 <!ENTITY insertMacro.label "Quick Bar">
 
-<!ENTITY pbMacroSearchPanel.firstrunWin "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Options > Composition > Advanced">
-<!ENTITY pbMacroSearchPanel.firstrunMac "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Preferences > Composition > Advanced">
+<!ENTITY pbMacroSearchPanel.firstrunWin "Bij het intypen van een URL vult Postbox deze automatisch aan tegen de URLs uit uw e-mails. Om URL automatisch aanvullen uit te schakelen gaat u naar Opties > Compositie > Geavanceerd">
```

Comment:
CORRECTIONS NEEDED. It's a literal translation. Although the words used are technically correct, it sounds off in Dutch. ''Tegen'' and ''Composition'' wouldn't be used in this context.